crcmocks/entitlements.py

```diff
@@ -1,28 +1,40 @@
 from flask import jsonify
 from flask import Blueprint
+from flask import request
+
+from crcmocks.config import DEFAULT_SERVICES
+from crcmocks.util import get_user_rh_identity
 
 
 blueprint = Blueprint("entitlements", __name__)
 
 
-services = [
-    "ansible",
-    "cost_management",
-    "insights",
-    "advisor",
-    "migrations",
-    "openshift",
-    "settings",
-    "smart_management",
-    "subscriptions",
-    "user_preferences",
-]
-entitlements = {}
-for svc in services:
-    entitlements[svc] = {"is_entitled": True, "is_trial": False}
-
-
-# TODO: allow customizable entitlements response per user
+# TODO: support is_trial?
 @blueprint.route("/v1/services")
 def services():
-    return jsonify(entitlements)
+    # get the identity header
+    identity_header = request.headers.get("X-Rh-Identity")
+
+    if not identity_header:
+        return "No identity header present in request\n", 401
```

Comment:
Do we really want the new lines in all the messages we return? (Here and below)

Reply:
When I was testing with `curl`, the `\n` made a new line for me in the console. I can remove if it preferred though.